docs/v2/data-sources/resource_integrations.md

```diff
@@ -128,6 +128,8 @@ Read-Only:
 - `attribute_type` (String) Type of the owner attribute (e.g., user, group).
 - `attribute_values` (List of String) List of assigned owner values.
 - `source_integration_name` (String) Name of the integration from which the type originates from (e.g. "Google Oauth").
+- `type` (String, Deprecated) Use `attribute_type` instead. `type` will be removed in v2.0.0.
```

Comment:
this shows that the only change in the schema... is that the two deprecated fields have been added to the datasource.